test/cpp/ext/otel/otel_tracing_test.cc

```diff
@@ -57,23 +63,35 @@ class OTelTracingTest : public ::testing::Test {
                 opentelemetry::exporter::memory::InMemorySpanExporterFactory::
                     Create(data_)));
     tracer_ = tracer_provider->GetTracer("grpc-test");
-    ASSERT_TRUE(OpenTelemetryPluginBuilder()
-                    .SetTracerProvider(std::move(tracer_provider))
-                    .SetTextMapPropagator(
-                        grpc::experimental::MakeGrpcTraceBinTextMapPropagator())
-                    .BuildAndRegisterGlobal()
-                    .ok());
+    ASSERT_TRUE(
```

Comment:
Please make sure that if these assertions fail, we print the failing status.

If `ASSERT_OK()` is available in OSS, that will do it for us.  Alternatively, we should do something like this:

```
absl::Status status = ...;
ASSERT_TRUE(status.ok()) << status;
```

Reply:
done